src/helpers/isActiveUser.js

```diff
@@ -0,0 +1,16 @@
+import { User } from '../queries';
+
+/**
+ * @param {object} input
+ * @returns {boolean} return true if the user is active or false if not
+ */
+export default async (input = {}) => {
+  const checkUser = await User.findOne(input);
+  if (!checkUser.errors && Object.keys(checkUser).length > 0) {
+    if (checkUser.isActive) {
+      return true;
+    }
+    return false;
```

Comment:
remove this return

Reply:
you mean to remove return false or all of them

Reply:
resolved